doc/tests/manual/reports/230725.md

```diff
@@ -0,0 +1,292 @@
+# Manual Test Report (pre v0.3 case study release)
+
+## General
+
+- Tester: Aydan Namdar Ghazani
+- Date/Time: 25 July 2023
+- Duration: 15 minutes
+- Commit/Tag: [af6b307](https://github.com/ElektraInitiative/PermaplanT/tree/af6b3070d285f273e6c173236a3c634eb4c827c3)
+- Planned tests: **13**
+- Executed tests: **13**
+- Passed tests: **10**
+- Failed tests: **3**
+
+## [Error Analysis](../README.md#report-header)
+
+- TC-006: Might be a faulty testcase.
+
+- TC-008: The test case worked, just the image loading error appeared.
+
+- TC-009: Same as 008.
+
+- TC-011: Functional deviation or limitation. Priority: TBD.
+
+- TC-012: Functional deviation or limitation. Priority: TBD.
```

Comment:
I don't think we need to prioritize in the test reports. But we should create issues and link them in the report.

Reply:
It would be good to know for the developers what kind of attention this problem needs.